src/components/domain-role/form/index.tsx

```diff
@@ -52,7 +52,7 @@ export function DomainRoleForm() {
     role: yup.string().required("Permissão é obrigatório"),
   });
 
-  const onSubmit = (values: DomainRole) => {
+  const onSubmit = (values: DomainRole/*, { setSubmitting }: FormikHelpers<DomainRole>*/) => {
```

Comment:
Remover comentários se não for mais necessário, verificar nos demais lugares com a mesma situação.

Reply:
Já foram removidos.